### ISSUE TITLE
Icon isseus

### ISSUE THREAD

**Issue by Huyeng** [oeco/opendev-wptheme#19]
Viewing the ODM website on Firefox, the icon of ODM logo and the three icons of Introductions don't not appear. Look at the screenshot below.
![icon_errors](https://cloud.githubusercontent.com/assets/6613530/6056326/ace24c72-ad41-11e4-93f6-1e3ac60f6659.png)

**Comment by acorbi** [comment]
I cannot reproduce this (using FF 35.0.1). For me, the icons are being shown properly.

**Comment by miguelpeixe** [comment]
This problem is possibly related to the site's url. When using `www` or not using it (depends on your actual URL), may cause some errors while viewing contents.

**Comment by miguelpeixe** [comment]
Any update on this issue @acorbi @Huyeng ?

**Comment by acorbi** [comment]
Exactly, when browsing this site **without www** icons are displayed properly.
Can this have something to do with the password protection currently active on the site?

![bildschirmfoto 2015-03-02 um 09 24 28](https://cloud.githubusercontent.com/assets/384894/6437341/f2ac14ec-c0bd-11e4-999e-163c6f369cdf.png)

**Comment by mishari** [comment]
I would like to confirm @acorbi's observation, I think we can close this issue.
